product/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+class ProductSize(models.Model):
+    size      = models.CharField(max_length=10)
+
+    class Meta:
+        db_table = 'product_sizes'
+
+class Product(models.Model):
+    name         = models.CharField(max_length=50)
+    price        = models.DecimalField(max_digits=20, decimal_places=4)
+    created_at   = models.DateTimeField(auto_now_add=True)
+    guide        = models.CharField(max_length=500, null=True)
+    product_size = models.ForeignKey('ProductSize', on_delete=models.CASCADE)
+    type_name    = models.ForeignKey('menu.TypeName', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'products'
+
+class Color(models.Model):
+    name    = models.CharField(max_length=100)
+    product = models.ManyToManyField('Product', through = 'ProductColor')
+
+    class Meta:
+        db_table = 'colors'
+
+class ProductColor(models.Model):
+    color   = models.ForeignKey('Color', on_delete=models.CASCADE)
+    product = models.ForeignKey('Product', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'product_colors'
+
+class ProductImage(models.Model):
+    image         = models.CharField(max_length=400)
```

Comment:
```suggestion
    image_url      = models.URLField(max_length=2000)
```
image_url로 해주세요. url정보를 저장할땐 URLField로 정의해주세요. 길이도 최소 2000 주세요.